src/js/view/controls/templates/settings/auto-quality.js

```diff
@@ -0,0 +1,5 @@
+export default () => {
+    return (
+        `<span class="jw-reset jw-auto-label"></span>`
```

Comment:
Shouldn't this just be added to the parent template? A module for this span is a bit much. The span can always be present in the menu as long as it's empty when there is no auto label.